game/building/src/index.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import * as React from 'react';
+import * as ReactDom from 'react-dom';
+import {createStore, applyMiddleware} from 'redux';
+import {Provider} from 'react-redux';
+var thunk = require('redux-thunk').default;
+
+import reducer from './services/session/reducer';
+import App from './components/BuildingApp';
+
+let store = createStore(reducer, applyMiddleware(thunk));
```

Comment:
Const not let [https://github.com/airbnb/javascript#references--prefer-const](prefer-const)